### PR TITLE
feat(msg error over ) : Improve error msg GO111MODULE=on

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
Just improve msg by adding a link with a further explanation in order to help the users. 

**Motivation for the change:**
https://github.com/operator-framework/operator-sdk/issues/1732 - Just improve the msg in order to by adding the link for the docs in order to help the users with.

### CODE REVIEW

**Comment by camilamacedo86** [comment]
c/c @hasbro17

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Document new compile-time dependency `mercurial` in user-facing documentation. ([#1683](https://github.com/operator-framework/operator-sdk/pull/1683))
 - Adds new flag `--zap-time-encoding` to the flagset provided by `pkg/log/zap`. This flag configures the timestamp format produced by the zap logger. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) for more information. ([#1529](https://github.com/operator-framework/operator-sdk/pull/1529))
+- Improve error message regards dependency manager `modules` requires env var GO111MODULE=on ([#1776](https://github.com/operator-framework/operator-sdk/pull/1776))
```
Comment (hasbro17) [inline CHANGELOG.md]: Probably better to leave this out of the CHANGELOG.
The CHANGELOG is more suited for user facing API/CLI/behavioral/workflow changes.

Comment (camilamacedo86) [reply CHANGELOG.md]: 👍

**Pull request merged**